ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -682,6 +789,15 @@ objects:
             exit 1
         fi
 
+        if [[ "${ENABLE_PROXY}" == "true" ]]; then
+          cat >> /tmp/artifacts/installer/install-config.yaml << EOF
+        proxy:
+          httpsProxy: https://admin:admin@${JOB_NAME_SAFE}-egress-proxy-${NAMESPACE}.svc.ci.openshift.org:443/
+          httpProxy: https://admin:admin@${JOB_NAME_SAFE}-egress-proxy-${NAMESPACE}.svc.ci.openshift.org:443/
```

Comment:
no need for the explicit `:443` since we're using `https://` for both of these.

Reply:
strangely enough, i found that when the `:443` was omitted it didn't correctly route to the proxy